### PR TITLE
changes node-recurly module

### PR DESCRIPTION
**Pull request by cgerrior**
@bhelx I went with the repo here: https://github.com/cgerrior/node-recurly (fork of https://github.com/breadbaker/node-recurly) as there are confirmed reports of it working. I also added a User-Agent string in "my" node-recurly.

### CODE REVIEW

File: api/node/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## API example: Node + Express
 
 This small application demonstrates how you might set up a web server
-using Ruby and [Express][express] with RESTful routes to accept your Recurly.js
+using Node.js and [Express][express] with RESTful routes to accept your Recurly.js
```
Comment (bhelx) [inline api/node/README.md]: Ha, nice catch

**Comment by bhelx** [comment]
:+1:

**Comment by bhelx** [comment]
@chrissrogers Can you merge please? Currently don't have permissions.

**Comment by chrissrogers** [comment]
:hamburger: :palm_tree:

**Pull request merged**